opuscleaner/filters/fix_un_chinese.py

```diff
@@ -8,7 +8,6 @@
 
 
 for line in sys.stdin:
-    line = line[:-1] #EoL
     line = line.strip()
```

Comment:
should this be `rstrip('\r\n')`

Reply:
I don't think that makes sense, everything `rstrip(\r\n)` would remove would also get nuked by strip() on the following line anyway.

Reply:
Ok, having read the documentation, i get it now.